5.sc_Morphology_Logistic_Regression/0.split_data/0.split_data_binary_classification.ipynb

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        random_state=1,
</pre><p>Consider changing this to zero per lab guidelines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I keep forgetting to set to 0. Nice catch!</p>